src/kata7.getEmployerRole.js

```diff
@@ -1,5 +1,13 @@
 const getEmployerRole = (employeeName, employees) => {
+  const nameArray = employeeName.map(function(x) {
+    return x.role
+ });
+
+
+
+return getEmployerRole('Javid', employees);
```

Comment:
You shouldn't know the employee name inside of the function definition, so should instead use the `employeeName` variable in place of `'Javid'`.